src/WinRT.Runtime/Projections.cs

```diff
@@ -25,9 +25,33 @@ namespace WinRT
 #endif
     static class Projections
     {
+        internal enum UiXamlMode
+        {
+            MicrosoftUiXaml,
+            WindowsUiXaml
+        }
+
+        private static UiXamlMode GetUIXamlModeSetting()

```

Comment:
You should be able to replace all this with `FeatureSwitches` now that that's in place. Should be very easy also to make everything trimmable, basically just minor changes than eg. #1446.